docs/kubectl-cost.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
@michaelmdresser do you think this should go here or under kubectl-cost?

Reply:
I think this should go under kubectl-cost!